packages/url_launcher/url_launcher_web/lib/url_launcher_web.dart

```diff
@@ -34,20 +34,25 @@ class UrlLauncherPlugin extends UrlLauncherPlatform {
     UrlLauncherPlatform.instance = UrlLauncherPlugin();
   }
 
-  String _getUrlScheme(String url) => Uri.tryParse(url)?.scheme;
+  String _getUrlScheme(String url) =>
+      Uri
+          .tryParse(url)
+          ?.scheme;
 
   bool _isSafariTargetTopScheme(String url) =>
       _safariTargetTopSchemes.contains(_getUrlScheme(url));
 
-  /// Opens the given [url] in a new window.
+  /// Opens the given [url] in the specified [webOnlyLinkTarget].
   ///
   /// Returns the newly created window.
   @visibleForTesting
-  html.WindowBase openNewWindow(String url) {
+  html.WindowBase openNewWindow(String url,{webOnlyLinkTarget} ) {
```

Comment:
Let's call the parameter:

`webOnlyWindowName` everywhere.

Also: this is not correctly `dartfmt`, so the formatting tests are failing.